src/QsCompiler/Transformations/Monomorphization.cs

```diff
@@ -52,7 +52,7 @@ public static QsCompilation Apply(QsCompilation compilation)
             var globals = compilation.Namespaces.GlobalCallableResolutions();
 
             var exemptCallableSet = globals
```

Comment:
Since this doesn't contain all the exempt callables anymore, it might be more appropriate to rename this, and associated variables/fields as `intrinsicCallableSet` to match the monomorphization validation.

Reply:
done